test/integration/azure/TESTING_AGAINST_AZURE.md

```diff
@@ -0,0 +1,44 @@
+# Testing Against Azure - Integration Testing
+
+## Problem Statement
+
+We want to test Azure-related InSpec resource against Azure itself. This requires a test fixture in Azure to exmine using InSpec.
+
+## General Approach
+
+We have a Terraform plan to setup and destroy test fixtures in Azure. When the environment is running we have a set of integration tests that may run against it.
+
+Credentials are handled via a `~/.azure/credentials` file.
+
+### Installing Terraform
+
+Download [Terraform](https://www.terraform.io/downloads.html).  We require at least v0.10 . To install and choose from multiple Terraform versions, consider using [tfenv](https://github.com/kamatama41/tfenv).
+
+## Current Solution
+
+In Azure we have setup an application. We use the credentials from this application to connect to Azure. From there we use Terraform to create the needed resources that we test against.
```

Comment:
"In Azure we have setup an application." →"We have set up an application in Azure."